### DISCUSSION TITLE
Is there a possibility to run compiled .deb from ubuntu v20.04 to install and run on ubuntu v22.04?

### DISCUSSION THREAD

**Discussion by thy-wilro** [tauri-apps/tauri#5393] [Q&A]
I tried to build .deb files on ubuntu 20.04 in tauri using "npm run tauri build", but when I try to install the .deb package from ubuntu 20.04 to install it on ubuntu 22.04, the result is successfully installed, but if I open the app to run, it won't open or not running in the background process, how can I able to fix this? or is there a way I can compile/build from ubuntu 20.04 and run it on ubuntu 22.04 or vice versa?

**Comment by FabianLars** [comment]
This is indeed the recommended/intended way (building on the oldest system you want to support). Can you start the app from inside a terminal and see if it prints any errors?

If there are no errors we need a little more info or a reproduction example.

**Reply by thy-wilro** [reply]
![image](https://user-images.githubusercontent.com/105403533/195562133-3bb6937a-f721-4852-a822-0a716c017d66.png)

This is what it looks like when I prompt to open in Terminal. It's from Ubuntu V22.04 compilation and trying to run on Ubuntu V20.04

What should I do in this? I'll try to search it too and find what solution it may be. :)
